src/main/java/appeng/client/gui/AEBaseGui.java

```diff
@@ -933,4 +927,29 @@ public void func_146977_a( final Slot s )
 	{
 		this.drawSlot( s );
 	}
+
+	protected GuiScrollbar getScrollBar()
+	{
+		return this.myScrollBar;
+	}
+
+	protected void setScrollBar( final GuiScrollbar myScrollBar )
```

Comment:
Might be an idea (for both). Also an example to prefer getter/setter internally. If a subclass overrides them, it will always use the correct ones.